Resources/views/Form/formatter.html.twig

```diff
@@ -99,7 +99,9 @@
             {% elseif format == 'rawhtml' %}
                 elms.markItUp(markitup_sonataHtmlSettings);
             {% elseif format == 'richhtml' %}
-                {{ ckeditor_widget(form.vars.id, ckeditor_configuration) }};
+                {{ ckeditor_widget(form.vars.id, ckeditor_configuration, {
+                    input_sync: true,
```

Comment:
I think this is BC and harmless, see http://symfony.com/doc/current/bundles/IvoryCKEditorBundle/usage/textarea-sync.html